src/Covid19Dashboard/ChartCarousel/ChartCarousel.less

```diff
@@ -45,7 +45,7 @@
 /** override some style for carousel's arrow and dots **/
 
 .slick-dots li button:before {
-  font-size: 20px;
+  font-size: 1.25rem;
```

Comment:
I didn't check all the px<-->rem conversions but the page LGTM, are you sticking to the baseline of `16px == 1rem`?

Reply:
yes that's right!